### PR TITLE
change php-cs-fixer to phpcs

### PR DESCRIPTION
**Pull request by DavidBadura**

### CODE REVIEW

File: phpcs.xml.dist [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<ruleset name="patchlevel/enum coding standard">
```
Comment (DanielBadura) [inline phpcs.xml.dist:2]:
```suggestion
<ruleset name="patchlevel/event-sourcing coding standard">
```

File: src/Aggregate/AggregateChanged.php [hunk 2]
```diff
@@ -75,8 +81,8 @@ public function recordNow(int $playhead): self
      */
     public static function deserialize(array $data): self
     {
-        /** @var string $class */
         $class = $data['event'];
+        assert(is_string($class));
```
Comment (DanielBadura) [inline src/Aggregate/AggregateChanged.php:85]: could we use webmozart/assert ?

Comment (DavidBadura) [reply src/Aggregate/AggregateChanged.php]: the phpcs fixer did that, I have now solved it with an exception. i don't want to add a library for one check.

**Comment by DavidBadura** [comment]
I have exclude SuperfluousSuffix for now, we can refactor the exceptions in another PR.

**Review by DanielBadura — APPROVED** [review]

**Pull request merged**